.teatro.yml

```diff
@@ -1,10 +1,10 @@
 stage:
   before:
-    - npm install -g npm
-    - cd frontend;
```

Comment:
afaik you don't need to enter the dir and leave it afterwards. Just for your info. Leave it as is for now :smile: